HopsanGUI/GUIObjects/GUIModelObject.cpp

```diff
@@ -1398,6 +1407,11 @@ QAction *ModelObject::buildBaseContextMenu(QMenu &rMenu, QGraphicsSceneContextMe
         mpParentSystemObject->getUndoStackPtr()->newPost();
         this->flipHorizontal();
     }
+    else if (selectedAction == pAlwaysVisbleAction)
+    {
+        mpParentSystemObject->getUndoStackPtr()->newPost();
```

Comment:
Not sure if `mpParentSystemObject->getUndoStackPtr()->newPost();` is needed. Since no undo data seems to be added ?

Reply:
I solved it the proper way instead, and added an undo post.